manifests/g/GaetanSencie/Comptabiliteannuelle/1.7.0.0/GaetanSencie.Comptabiliteannuelle.locale.fr-BE.yaml

```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: GaetanSencie.Comptabiliteannuelle
+PackageVersion: 1.7.0.0
+PackageLocale: fr-BE
+Publisher: Gaëtan Sencie
+PublisherUrl: https://github.com/STENS66
+PublisherSupportUrl: https://github.com/STENS66/Comptabilite_annuelle/issues
+PackageName: Comptabilité annuelle
+PackageUrl: https://github.com/STENS66/Comptabilite_annuelle
+License: Property
+Copyright: © Gaëtan Sencie 2023 tous droits réservés
+ShortDescription: Comptabilité annuelle Setup
```

Comment:
Please provide a better description of what this package is, what it does, or what it is used for